packages/design-system/src/components/Tooltip/Tooltip.stories.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Tooltip from './Tooltip';
+import TooltipIcon from './TooltipIcon';
+import Button from '../Button/Button';
+
+export default {
+  title: 'Components/Tooltip',
+  component: Tooltip,
+  argTypes: {
+    children: { control: false },
+    data: { table: { disable: true } },
+    placement: {
+      options: ['auto', 'left', 'right', 'top', 'bottom'],
+      control: { type: 'select' },
+    },
+  },
+};
+
+const Template = ({ data, ...args }) => (
+  <div className="ds-u-display--flex ds-u-align-items--center ds-u-margin-x--4 ds-u-margin-y--3">
+    {data}
+    <Tooltip {...args} />
+  </div>
+);
```

Comment:
I noticed that since the tooltip trigger is near the top of the window, there's no way to actually see the `top` positioning. Even if you select `top`, it Popper will never render it on the top because there's no room. I like that it's smart like that, but maybe our example should change to allow people to see that option.

Reply:
Good call, updated to add more spacing, can now see top.